### ISSUE TITLE
CTRL+F (browser "search in page") is broken on Linux Firefox

### ISSUE THREAD

**Issue by JobLeonard** [observablehq/feedback#110]
**Describe the bug**

If I press CTRL+F on Firefox to open the "search in page" field it instead advances the caret by one step if a cell is selected

**To Reproduce**
1. Open a cell, make sure it has focus and the caret
2. press CTRL+F (which should open the "find in page" functionality)

**Expected behavior**

"Find In Page..." to be opened

**Desktop (please complete the following information):**

Linux Firefox 89

**Comment by CobusT** [comment]
This is in 'experimental features'

**Comment by JobLeonard** [comment]
So I found that one way to make the find-in-page open again is to first click on the right side of a cell, like where I pointed my cursor in the screenshot:

![image](https://user-images.githubusercontent.com/259840/123060237-d6985100-d40a-11eb-94e5-1000b45e29c6.png)

After doing that CTRL+F works as expected. Does a cell having focus perhaps capture the keyboard input in a way that intervenes? 

However, even though we can open the find-in-page this way, the search behavior is very unreliable: it can't really "forward" search, only "backwards", and even then it add/skips items half of the time.

https://user-images.githubusercontent.com/259840/123060025-a5b81c00-d40a-11eb-88c7-b56dd5754d6b.mp4

Note how the number of found matches toggles between five and six. Chromium-based browsers show the correct seven matches without any hassle:
![image](https://user-images.githubusercontent.com/259840/123061109-9f766f80-d40b-11eb-8293-822ead66311a.png)

(remember, this issue shows up in Firefox, Chromium-based browsers don't seem to have this issue - I can't speak for any other browsers than those two)

**Comment by mootari** [comment]
Related: https://github.com/observablehq/feedback/issues/84

**Comment by CobusT** [comment]
We now have a dedicated Find and Replace feature for notebooks
https://observablehq.com/@observablehq/release-notes#cell-957

**Comment by lionel-rowe** [comment]
Why's this closed? It seems like Ctrl+F is still overridden to be "move cursor forward 1 step" (with Ctrl+B being "move cursor back 1 step"), but would anyone ever want that feature? Surely that's what arrow keys are for. I'd much rather have it remain as the native browser find-in-page feature, which is sometimes desirable to use over the custom Find-and-Replace (e.g. I want to search the literal displayed text content of the page). I know I can also use F3, but Ctrl+F is more convenient.

**Comment by CobusT** [comment]
Agreed. It should not have been closed.

**Comment by mootari** [comment]
@lionel-rowe Do you see the same override and behavior in the CodeMirror demo editor on https://codemirror.net/ ?

**Comment by lionel-rowe** [comment]
> @lionel-rowe Do you see the same override and behavior in the CodeMirror demo editor on https://codemirror.net/ ?

No. For me (Chrome 119, Windows 11), that demo editor does override Ctrl+F, but the new functionality is the code editor's own find-and-replace, rather than moving the cursor.
